Examples/Scripts/ApplyAutoRecleanPipelineBatch.sh

```diff
@@ -0,0 +1,194 @@
+	then
+		echo "ERROR: EnvironmentScript not specified"
+		exit 1
+	fi
+
+	if [ -z ${RunLocal} ]
+	then
+		echo "ERROR: RunLocal is an empty string"
+		exit 1
+	fi
+
+	# report options
+	echo "-- ${scriptName}: Specified Command-Line Options: -- Start --"
+	echo "   StudyFolder: ${StudyFolder}"
+	echo "   Subjlist: ${Subjlist}"
+	echo "   EnvironmentScript: ${EnvironmentScript}"
+	echo "   RunLocal: ${RunLocal}"
+	echo "-- ${scriptName}: Specified Command-Line Options: -- End --"
+}  # get_options()
+
+#
+# Function Description
+#	Main processing of this script
+#
+#	Gets user specified command line options and runs a batch of ReApplyFix processing
+#
+main() {
+	# get command line options
+	get_options "$@"
+
+	# set up pipeline environment variables and software
+	source "$EnvironmentScript"
+
+	# set list of fMRI
+	fMRINames="rfMRI_REST1_RL@rfMRI_REST1_LR@rfMRI_REST2_LR@rfMRI_REST2_RL"
+  
+	# specify the name of concatenated folder
+	# if run Multi-Run specify ConcatNames as null string
+	MRConcatfMRIName="rfMRI_REST"
+
+	# set highpass
+	highpass=2000
+	
+	# set resolution
+	fMRIResolution="2"
+
+	# set singularity path
+	PythonSingularity="/path/to/singularity.img"
+	PythonSingularityMountPath="/path/to/data" # where the data dir need to mount to the singularity
+
+	# # set python interpreter
+	# PythonInterpreter="/my/conda/path/envs/hcp_python_env/bin/python3"
+	PythonInterpreter=""
+	
+	# models to use
+	Models="RandomForest@MLP@Xgboost@WeightedKNN@XgboostEnsemble"
+
+	# set threshold
+	VoteThresh="5" # the number stands for how many votes to finalze the classification, if there are 5 models, then vote threshold 5 means only reclassify when the 5 models all agree on the prediction
+
+	#NOTE: syntax for QUEUE has changed compared to earlier pipeline releases,
+	#DO NOT include "-q " at the beginning
+	#default to no queue, implying run local
+	QUEUE=""
+	#QUEUE="long.q"
+
+	# low res mesh
+	LowResMesh=32
+
+	if [[ "${RunLocal}" == "TRUE" || "$QUEUE" == "" ]]; then
+		queuing_command=("$HCPPIPEDIR"/global/scripts/captureoutput.sh)
+	else
+		queuing_command=("${FSLDIR}/bin/fsl_sub" -q "$QUEUE")
+	fi
+
+	for Subject in ${Subjlist}; do
+		# if run via singularity
+		"${queuing_command[@]}" "$HCPPIPEDIR"/ICAFIX/ApplyAutoRecleanPipeline.sh \
+		--study-folder="$StudyFolder" \
+		--subject="$Subject" \
+		--fmri-names="$fMRINames" \
+		--mrfix-concat-name="$MRConcatfMRIName" \
+		--fix-high-pass="$highpass" \
+		--fmri-resolution="$fMRIResolution" \
+		--subject-expected-timepoints="$subjectExpectedTimepoints" \
+		--low-res="$LowResMesh" \
+		--python-singularity="$PythonSingularity" \
+		--python-singularity-mount-path="$PythonSingularityMountPath" \
+		--python-interpreter="$PythonInterpreter" \
+		--model-to-use="$Models" \
+		--vote-threshold="$VoteThresh"
+
+		# # instead of running via singularity, one can also run via native enviroment enabled by conda
+		# # you must specify the $HCPCONDAENV in the env script to source
```

Comment:
This method doesn't need an env variable anymore, right?  So this comment should be removed?